signer/src/transaction_coordinator.rs

```diff
@@ -387,6 +392,63 @@ where
         .map(|_| ())
     }
 
+    async fn construct_and_send_bitcoin_presign_request(
+        &mut self,
+        bitcoin_chain_tip: &model::BitcoinBlockHash,
+        pending_requests: &SbtcRequests,
+        transaction_package: &[utxo::UnsignedTransaction<'_>],
+    ) -> Result<(), Error> {
+        // Create the BitcoinPreSignRequest from the transaction package
+        let sbtc_requests = BitcoinPreSignRequest {
+            requests: transaction_package
+                .iter()
+                .map(|tx| (&tx.requests).into())
+                .collect(),
+            fee_rate: pending_requests.signer_state.fee_rate,
+            last_fees: pending_requests.signer_state.last_fees.map(Into::into),
+        };
+
+        // Define the filter for presign request received events
+        let presign_ack_filter = |event: &SignerSignal| -> bool {
+            matches!(
+                event,
+                SignerSignal::Event(SignerEvent::TxSigner(
+                    TxSignerEvent::BitcoinPreSignRequestReceived
+                ))
+            )
+        };
+
+        // Create a signal stream with the defined filter
+        let signal_stream = self.context.as_signal_stream(presign_ack_filter);
+
+        // Send the presign request message
+        self.send_message(sbtc_requests, bitcoin_chain_tip).await?;
+
+        tokio::pin!(signal_stream);
+        let future = async {
+            let mut num_acks = 1;
+            while num_acks < self.threshold {
+                // If signal_stream.next() returns None then one of the
+                // underlying streams has closed. That means either the
+                // network stream, the internal message stream, or the
+                // termination handler stream has closed. This is all bad,
+                // so we trigger a shutdown.
+                let Some(_) = signal_stream.next().await else {
+                    tracing::warn!("signal stream returned None, shutting down");
+                    self.context.get_termination_handle().signal_shutdown();
+                    return Err(Error::SignerShutdown);
+                };
+                num_acks += 1;
+            }
+            Ok(())
+        };
```

Comment:
At the moment they are simple ACKs. But, they can be easily extended to return an ordered vector of bool representing the will to vote per each request, in the same order as they were received.